src/libs/actions/Policy.ts

```diff
@@ -609,7 +606,14 @@ function setWorkspaceApprovalMode(policyID: string, approver: string, approvalMo
         },
     ];
 
-    const params: SetWorkspaceApprovalModeParams = {policyID, value: JSON.stringify(value)};
+    const params: SetWorkspaceApprovalModeParams = {
+        policyID,
+        value: JSON.stringify({
+            ...value,
+            // This property should now be set to false for all Collect policies
+            isAutoApprovalEnabled: false,
```

Comment:
discussed this and we actually want the autoapproval to be off for Collect policies